step-4.json

```diff
@@ -17,11 +17,23 @@
   "flop_tau": "172800",
   "tokens": {
     "ETH": {
-      "pip": "0x8C73Ec0fBCdEC6b8C060BC224D94740FD41f3774",
+      "pip": {
+        "osmDelay": "0",
+        "type": "value",
+        "price": "150",
+        "signers": [
```

Comment:
if `type` is `value`, do we need `signers`?

Reply:
For this purpose no, but price contract will eventually become a user configurable option, each config file has properties for both. Right now I believe everything defaults to `value`.